src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.const.tsx

```diff
@@ -74,12 +74,10 @@ export const itemsTypeOptions: ItemsOptionGroup[] = [
       {
         value: ItemResponseType.SingleSelectionPerRow,
         icon: itemsTypeIcons[ItemResponseType.SingleSelectionPerRow],
-        isMobileOnly: true,
```

Comment:
If I'm not mistaken we have some unit tests for the ``isMobileOnly`` chip. Could you please add these cases to them?